### ISSUE TITLE
About the ImageNet pretrained checkpoints.

### ISSUE THREAD

**Issue by PkuRainBow** [NVIDIA/partialconv#2]
Really simple and interesting work.

I am wondering when it will be convenient for you to share the imagenet pretrained models.

I hope to try your pd_resnet on the scene parsing tasks.

**Comment by liuguilin1225** [comment]
If you will train (fine-tune) the resnet backbone during training your scene parsing network instead of fixing it, you can directly using the existing model pretrained with zero padding.  Just change your nn.Conv2d to PartialConv2d. Their parameter numbers are the exactly same. 
As long as you fine-tune it enough, it should not matter too much. This is also reflected in Kaiming's recent paper:  Rethink ImageNet Pre-training. https://arxiv.org/abs/1811.08883
The segmentation experiments  in the paper are also both using the pretrained weights with zero padding; but during the fine-tuning, one is using nn.Conv2d while the other is using PartialConv2d.
I will upload the pretrained weights of partial conv powered models soon.

**Comment by PkuRainBow** [comment]
@liuguilin1225 Thanks. I will try it latter.

**Comment by liuguilin1225** [comment]
@PkuRainBow The pretrained weights for VGG and ResNet networks with partial convolution based padding can be found here: https://www.dropbox.com/sh/t6flbuoipyzqid8/AACJ8rtrF6V5b9348aG5PIhia?dl=0
